package.json

```diff
@@ -90,5 +90,8 @@
     "vue-template-compiler": "^2.6.10",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.3.10"
+  },
+  "dependencies": {
```

Comment:
typescript is an option for who likes to use it only, not a dependency of this plugin

Reply:
Yeah, see my other comment which describes how `@types/gtag.js` is used.

With that said, I have not actually referenced types like this for the purpose of a type declaration file and would be fine just copying some of the `@types/gtag.js` types into this project's declaration file, removing this dependency.